### DISCUSSION TITLE
Using django's internationalization (i18n)

### DISCUSSION THREAD

**Discussion by andreighh** [silviogutierrez/reactivated#355] [Q&A] [answered]
Is there any way to leverage django's i18n? I know about https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#internationalization-in-javascript-code but I'm curious if it applies here or if there are any other methods?

Thank you for the time and project!

**Comment by silviogutierrez** [comment, marked as answer]
Since most things are rendered by React templates, I don't think it's possible. 

However, that also means you can use the React ecosystem. And that may be even more powerful:

https://react.i18next.com
https://formatjs.io/docs/react-intl/

**Reply by andreighh** [reply]
Thank you! I was hoping I could use something like `django-rosetta` (https://django-rosetta.readthedocs.io/) so translators could translate directly in the admin without having to go through the PO file / JSON with a PO editor. I guess I could make an UI for it and keep the JSONs in the database.

With that being said, I consider using this library for a big upcoming project I'll have. Is it ok if I ask a few more questions and if yes, how should I do it - continue here / make a new discussion per question / make a new discussion with multiple questions?

**Reply by silviogutierrez** [reply]
Sure, just look through discussions for dupes and also the docs. If still unanswered, just open a discussion and I'm happy to help.

If you run into major issues, we can do a screen share to diagnose.
